From a5acf91cc0040ef6c33b2de7f59d0124dee0fc4e Mon Sep 17 00:00:00 2001
From: Rocky Hao <rocky.hao@rock-chips.com>
Date: Tue, 27 Jun 2017 15:10:39 +0800
Subject: [PATCH] arm: dts: rockchip: rk3288: update tsadc's working frequence

tsadc in rk3288 can not work well at 10k hz clock. at most time, tsadc
in rk3288 can work well, but it will wrongly report a very high
temperature (eg. 110 degree) occasionally.  we set 5k hz can
resolve this issue.

Change-Id: I06a4a1631baa5f65786a524e2430bb2386f52bdf
Signed-off-by: Rocky Hao <rocky.hao@rock-chips.com>
---
 arch/arm/boot/dts/rk3288.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/rk3288.dtsi b/arch/arm/boot/dts/rk3288.dtsi
index 8b920ac4bd75..88979ebdd380 100644
--- a/arch/arm/boot/dts/rk3288.dtsi
+++ b/arch/arm/boot/dts/rk3288.dtsi
@@ -572,7 +572,7 @@
 		clocks = <&cru SCLK_TSADC>, <&cru PCLK_TSADC>;
 		clock-names = "tsadc", "apb_pclk";
 		assigned-clocks = <&cru SCLK_TSADC>;
-		assigned-clock-rates = <10000>;
+		assigned-clock-rates = <5000>;
 		resets = <&cru SRST_TSADC>;
 		reset-names = "tsadc-apb";
 		pinctrl-names = "init", "default", "sleep";
-- 
2.35.3

